docs/config_reference.rst

```diff
@@ -1301,6 +1301,16 @@ General Configuration
   Timeout value in seconds used when checking if a git repository exists.
 
 
+.. js:attribute:: .general[].pipeline_statistics
+
+   A boolean value indicating whether we want to collect statistic information for the execution of the asynchronous pipeline.
```

Comment:
```suggestion
   Dump pipeline progress for the asynchronous execution policy.
   This is option is meant for debug purposes only.
```

You should also mention the file that it generates.

Generally, what we do here is not statistics, but rather dumping the progress of the tests in the pipeline.